src/framework/theme/components/calendar-kit/components/index.ts

```diff
@@ -6,8 +6,16 @@
 
 export { NbCalendarHeaderComponent } from './calendar-header/calendar-header.component';
 export { NbCalendarDayCellComponent } from './calendar-day-picker/calendar-day-cell.component';
-export { NbCalendarYearPickerComponent } from './calendar-year-picker/calendar-year-picker.component';
-export { NbCalendarMonthPickerComponent } from './calendar-month-picker/calendar-month-picker.component';
+export {
+  NbCalendarYearPickerComponent,
+  YEARS_IN_VIEW,
+  YEARS_IN_COLUMN,
+} from './calendar-year-picker/calendar-year-picker.component';
+export {
+  NbCalendarMonthPickerComponent,
+  MONTHS_IN_VIEW,
+  MONTHS_IN_COLUMN,
```

Comment:
Constants seem to be unused.

Reply:
It's fine since we exporting them. Resolving as discussed irl.